recipe/meta.yaml

```diff
@@ -16,22 +16,23 @@ source:
     - patches/0015-Don-t-build-redis-with-jemalloc-on-linux-aarch64.patch  # [linux and aarch64]
     - patches/0016-fix-redis-build-on-macos.patch  # [osx]
     - patches/0017-Don-t-use-Xcode-on-macOS.patch
+    - patches/0018-Remove-runtime-agent-dependencies.patch
 
 build:
-  number: 2
-  skip: true  # [py<38 or py>=312]
-  # skipping architectures that do not yet have bazel, so only build for linux64 & win64.
-  # skipping osx and aarch64 for now, as there are complicated bazel errors with external redis (symbols, dependant libraries, and objects).
+  number: 0
+  # Python 3.12 is excluded because of gymnasium 0.28.1
```

Comment:
I think @cbouss built/updated gymnasium recently.

Reply:
@JeanChristopheMorinPerso I re-enabled py3.12 but forgot to remove the comment. Thanks !

Reply:
Fixed